grid/p5js/chance_grid/index.md

```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/grid/p5js/chance_grid
+title: Grid
+tags: P5.js, "Law of Proximity", "Law of Connected-Elements", "Law of Proximity", "Multistable-perception" Chance, Grid
+permalink: /grid/Grid/
+summary: This sample shows a dynamic grid, that changes the width and height of itself.  
+---
+
+The grid changes the size of the rectangles that are built through the grid each time by clicking the updating button. It is a non symetric drawn horizontal and vertical line canvas, that makes it interesting for drawing sheet.  
+
+![](http://www2.pic-upload.de/img/32318238/grid.jpg)
```

Comment:
No hard coded URLS please. The canvas from the P5js sketch is added dynamically to the page when using the p5js layout